packages/styled-system/src/config/__tests__/border.test.js

```diff
@@ -155,3 +155,66 @@ test('returns border styles', () => {
     borderSpacing: '.125rem',
   });
 });
+
+describe('borderRadius with compoundThemeValueTransform', () => {
+  const theme = {
+    radii: {
+      sm: '0.125rem',
+      md: '0.25rem',
+      lg: '0.5rem',
+      xl: '1rem'
+    }
+  };
+
+  test('single value should apply to all corners', () => {
+    const style = border({
+      theme,
+      borderRadius: 'sm'
+    });
+    expect(style).toEqual({
+      borderTopLeftRadius: '0.125rem',
+      borderTopRightRadius: '0.125rem',
+      borderBottomLeftRadius: '0.125rem',
+      borderBottomRightRadius: '0.125rem'
+    });
```

Comment:
It is recommended to maintain the following key order for consistency of `border-radius` declaration:
```
borderTopLeftRadius
borderTopRightRadius
borderBottomRightRadius
borderBottomLeftRadius
```